src/main/java/org/minions/devfund/damianvp/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,134 @@
+            Arrays.stream(actors).forEach(actor -> {
+                Actor actorObject = getActorObject(actor);
+                int index = this.actorList.indexOf(actorObject);
+                this.actorList.get(index).addMovie(movie);
+                movie.addActor(this.actorList.get(index));
+            });
+            this.movieList.add(movie);
+        }
+    }
+
+    /**
+     * Method to check if movie already exist in class.
+     * @param name String type.
+     * @return true: if movie exist.
+     *         false: if movie does not exist.
+     */
+    public boolean containsMovie(final String name) {
+        return this.movieList.stream().anyMatch(o -> o.getName().equals(name));
+    }
+
+    /**
+     * Method to get the Actor object.
+     * @param name String type.
+     * @return Actor object.
+     */
+    public Actor getActorObject(final String name) {
+        Optional<Actor> actor = this.actorList.stream().filter(o -> o.getName().equals(name)).findAny();
+        if (actor.isPresent()) {
+            return actor.get();
+        }
+        Actor newActor = new Actor(name);
+        this.actorList.add(newActor);
+        return newActor;
+    }
+
+    /**
+     * Method to get the Movie object form the list by movie name.
+     * @param name String type.
+     * @return Movie object from list: if movie already exists.
+     *         new Movie object: if movie does not exists.
+     */
+    public Movie getMovieObject(final String name) {
+        return this.movieList.stream().findAny().filter(o -> o.getName().equals(name)).orElse(new Movie(name));
+    }
+
+    /**
+     * Method to add a rating to a specific movie.
+     * @param name String type, name of movie.
+     * @param rating double type, rating to set.
+     */
+    void addRating(final String name, double rating) {
+        for (int i = 0; i < this.movieList.size(); i++) {
+            if (this.movieList.get(i).getName().equals(name)) {
+                this.movieList.get(i).setRating(rating);
+                return;
+            }
+        }
+    }
+
+    /**
+     * Method to update the rating of a specific movie.
+     * @param name String type, name of movie.
+     * @param newRating double type, rating to update.
+     */
+    void updateRating(final String name, double newRating) {
+        addRating(name, newRating);
+    }
+
+    /**
+     * Method to get the nest actor of class.
+     * @return String type, name of the best actor.
+     */
+    String getBestActor() {
+//        Comparator<Actor> comparator = Comparator.comparing(Actor::getRatingAverage);
+        return this.actorList.stream().max(Comparator.comparing(Actor::getRatingAverage)).orElse(new Actor()).getName();
+    }
+
+    /**
+     * Method to get the best movie of the class.
+     * @return String type, name of best movie.
+     */
+    String getBestMovie() {
+//        Comparator<Movie> comparator = Comparator.comparing(Movie::getRating);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)